change/@fluentui-web-components-f70c1f20-47e3-4e76-9cb0-53fef171209f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29505&commitId=075c0fd40bb8eaecdc947fc60051150861139354&env=prod&clientType=FLUENTUI)


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
<img width="1104" alt="image" src="https://github.com/microsoft/fluentui/assets/13071055/f2fbeb7b-8bd0-4d1a-9d2a-6955992271de">

Reply:
Switch.Checked - RTL.normal.chromium.png
![image](https://github.com/microsoft/fluentui/assets/9615899/e5bcbedc-0026-4356-8c55-7961a319eddc)

Switch.Checked - RTL.hover.chromium.png
![image](https://github.com/microsoft/fluentui/assets/9615899/08578401-f0e3-42ea-80e9-c91787585fcf)